src/Extension/Tasks/BusinessModelInstallerTask.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Extension\Tasks;
+
+use SprykerSdk\Sdk\Core\Domain\Entity\Lifecycle\InitializedEventData;
+use SprykerSdk\Sdk\Core\Domain\Entity\Lifecycle\Lifecycle;
+use SprykerSdk\Sdk\Core\Domain\Entity\Lifecycle\RemovedEventData;
+use SprykerSdk\Sdk\Core\Domain\Entity\Lifecycle\UpdatedEventData;
+use SprykerSdk\Sdk\Core\Domain\Entity\Placeholder;
+use SprykerSdk\SdkContracts\Entity\Lifecycle\LifecycleInterface;
+use SprykerSdk\SdkContracts\Entity\TaskInterface;
+
+class BusinessModelInstallerTask implements TaskInterface
+{
+    /**
+     * @var array<\SprykerSdk\SdkContracts\Entity\CommandInterface>
+     */
+    protected array $commands = [];
+
+    /**
+     * @param array<\SprykerSdk\SdkContracts\Entity\CommandInterface> $commands
+     */
+    public function __construct(array $commands)
+    {
+        $this->commands = $commands;
+    }
+
+    /**
+     * @return string
+     */
+    public function getShortDescription(): string
+    {
+        return 'Install business model B2B or B2C';
+    }
+
+    /**
+     * @return array
+     */
+    public function getPlaceholders(): array
+    {
+        return [
+            new Placeholder(
+                '%business_model_url%',
+                'B2BC_TYPE',
+            ),
+            new Placeholder(
+                '%pc_system%',
```

Comment:
Let's use PCSystemValueResolver::ALIAS, what do you think?
The same for `B2BC_TYPE`, move this value to ValueResolver const and use here